src/ctap/mod.rs

```diff
@@ -261,14 +261,15 @@ fn parse_metadata(
 /// The public key is COSE encoded, and the hash is a SHA256.
 fn verify_signature(
     signature_bytes: &[u8; 64],
-    public_key_bytes: &[u8],
+    public_key_bytes: &[u8; 64],
     signed_hash: &[u8; 32],
 ) -> Result<(), Ctap2StatusCode> {
     let signature = ecdsa::Signature::from_bytes(signature_bytes)
         .ok_or(Ctap2StatusCode::CTAP1_ERR_INVALID_PARAMETER)?;
-    let cbor_public_key = cbor_read(public_key_bytes)?;
-    let cose_key = CoseKey::try_from(cbor_public_key)?;
-    let public_key = ecdsa::PubKey::try_from(cose_key)?;
+    let x = array_ref!(public_key_bytes, 0, 32);
+    let y = array_ref!(public_key_bytes, 32, 32);
+    let public_key = ecdsa::PubKey::from_coordinates(x, y)
+        .ok_or(Ctap2StatusCode::CTAP2_ERR_VENDOR_INTERNAL_ERROR)?;
```

Comment:
Looks like we could use `PubKey::from_bytes_uncompressed` and directly the 65 bytes in `build.rs`. No strong preference, but `from_bytes_uncompressed` would make both call sites smaller for just 1 more byte stored in the binary.

Reply:
Good idea, done. Let's check binary size. Currently:
```
OLD 115.2 kiB
NEW 115.0 kiB
```

Reply:
Still the same now.